tasks/ubuntu/bionic.task/os_complete.erb

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+echo Razor policy successfully applied > /tmp/razor_complete.log
+echo Installed using <%= task.label %> - <%= task.description %> >> /tmp/razor_complete.log
+echo Repo: <%= repo_url %> >> /tmp/razor_complete.log
+echo Node: <%= node_url %> >> /tmp/razor_complete.log
+
+sed -i '/razor_postinstall/d' /etc/crontab
```

Comment:
This looks like the only line that has changed so far. Was the other method of stopping the postinstall script not working?

Reply:
post install script was never called because ubuntu 18.04 uses systemd instead of init.

adding a line to the crontab works with systemd.

Reply:
Ah excellent, that makes sense. Is there a reason this wouldn't work for other Ubuntu versions too? This might be a more portable way to be standard across versions.

Reply:
I haven't tested this with other than ubuntu 18.04, this might work for newer version but not sure for older.